src/software/time/time.cpp

```diff
@@ -23,3 +20,27 @@ double Time::getMilliseconds() const
 }
 
 Time::~Time() {}
+
+double Time::now()
+{
+    const auto clock_now = std::chrono::system_clock::now();
+    return static_cast<double>(std::chrono::duration_cast<std::chrono::microseconds>(
```

Comment:
any reason why we do this over just `std::time(NULL)`? that gives us exactly what we want. just curious what benefit boost has here

Reply:
more precision. I think time(null) is only second precision. This gives us microsecond precision